src/server/helpers/config.js

```diff
@@ -9,7 +9,8 @@ import dotenv from 'dotenv';
  */
 const CLIENT_SIDE_WHITELIST = [
   'NODE_ENV',
-  'BASE_URL'
+  'BASE_URL',
+  'LP_API_USERNAME'
```

Comment:
It means that things wouldn't work during development unless you supplied Launchpad credentials.  That's fine by me (and I could make the errors in that case a little more graceful), but are you two OK with that?